zen-quotes-app/src/app/home/helpers.ts

```diff
@@ -0,0 +1,8 @@
+// @ts-ignore
+// @Todo
+//  @Ira: when we run `ng test`, the tests in helpers.spect.ts fail.
```

Comment:
@IraLysenko Some unit tests were added in helpers.spec.ts.

When you run run `ng test`, you will notice that some of them fail because the `makeAuthorsUnique` function is not implemented correctly.

Challenge: can you make the tests succeed? ⚡ 

Thanks and best of luck! 🍀

Reply:
@jvandemo
Challenge accepted 🙏 🙇
Thank you very much for your time!🙏

Reply:
Nice authors by the way 😊😊😊

Reply:
😂 😂 🙌